index.bs

```diff
@@ -691,7 +700,6 @@ string-valued keys. Values may be any type that has a valid encoding in JSON. It
     dictionary ClientData {
         required DOMString           challenge;
         required DOMString           origin;
-        required DOMString           rpId;
```

Comment:
See #189. The rpId is being passed in the options so it's not clear why it should also be in the client data. Also the RP ID now gets hashed into the signature as well so it seems redundant to have this here.